common.mk

```diff
@@ -0,0 +1,4 @@
+D = /
```

Comment:
Please add the copyright information before the first line, you can find an example from other makefiles.